registry/quilt_server/views.py

```diff
@@ -128,6 +135,7 @@ def __init__(self, name):
 
 ### Web routes ###
 
+# TODO: deprecate
```

Comment:
Actually, can you just delete all oauth2-related code - including vars in `prod_config.py`, etc?

I've already run into this when trying to deploy in CF: I didn't set any OAUTH_* vars, and got exceptions.